app/assets/javascripts/govuk_publishing_components/rum-loader.js.erb

```diff
@@ -0,0 +1,36 @@
+(function() {
+  var marker = document.getElementsByTagName('script')[0]
+  var scripts = [
+    '<%= path_to_javascript("govuk_publishing_components/vendor/lux/lux-polyfill") %>',
+    '<%= path_to_javascript("govuk_publishing_components/vendor/lux/lux") %>'
+  ]
+  var parsedCookie = (function() {
+    try {
+      var cookies = document.cookie.split(';')
+
+      for (var i = 0; i < cookies.length; i++) {
+        var cookie = cookies[i].replace(/\s/g, '') // Remove all whitespace.
+        var cookieName = 'cookies_policy='
+
+        if (cookie.indexOf(cookieName) === 0) {
+          return JSON.parse(cookie.substring(cookieName.length))
+        }
+      }
+    } catch (error) {
+      console.error(error)
```

Comment:
Probably out of scope for this PR, but I wonder if there's something better we can do in the event of an error, like firing a specific GA event? Would let us have some awareness of problems occurring.

Reply:
We can and I'll do some digging and see if this is feasible - but, as you say, out of scope for the moment. I think frontend error reporting might be a too big an issue to be solved here!